crates/language-tests/src/cmd/upgrade/process.rs

```diff
@@ -32,14 +32,6 @@ pub struct Failure {
 	pub message: String,
 }
 
-#[revisioned(revision = 1)]
-#[derive(Debug)]
-pub enum Data {
-	Other(Value),
-	Query(Vec<dbs::QueryMethodResponse>),
-	Live(dbs::Notification),
-}
-
```

Comment:
Removing this struct without changing the rest of the types breaks the upgrade test.